### ISSUE TITLE
Get RAW Data

### ISSUE THREAD

**Issue by maxp1256** [fosterbuster/iM871A-USB#10]
Hello!

How is it possible to get raw data from meter devices? There is no output regarding received data packets in the output. Shouldn't it?

**Comment by maxp1256** [comment]
Hello,
I've tried several things:
* Downloaded Wireless M-Bus Studio
* Set Device Mode to Other, [x] Store in Flash Memory
* Set Link Mode to T1, Channel 0 (?)
* Activated and Set Real Time Clock

Added to your Sample:
      await modem.TransmitMessage(new SetDeviceConfigurationRequest(DeviceMode.Other));
      await modem.TransmitMessage(new SetOperationModeRequest(SystemOperationMode.Default));

and now it's working.

I don't know what of these steps are exactly needed to get it working with a new stick (and I don't think that I would buy another one in next time), but that was all I've done and it's logging the data now to console. :-) Maybe this will help someone else in future!

A big thank you for your work, you've saved me a lot of time for my private water consumption project!
